library/tomee

```diff
@@ -1,67 +1,52 @@
-Maintainers: Alex Soto <asotobu@gmail.com> (@lordofthejars), Otavio Santana <otaviojava@apache.org> (@otaviojava), Jonathan Gallimore <jgallimore@apache.org> (@jgallimore)
+Maintainers: Alex Soto <asotobu@gmail.com> (@lordofthejars), Otavio Santana <otaviojava@apache.org> (@otaviojava), Jonathan Gallimore <jgallimore@apache.org> (@jgallimore), Rod Jenkins <jenkir14@nationwide.com> (@scriptmonkey)
 GitRepo: https://github.com/tomitribe/docker-tomee.git
-GitCommit: fbe01211d21051406ffaae5bdf41d1f629caf58f
+GitCommit: 042b7dbd684f9b3649b6d1b4b924f38dedf85c19
 Architectures: amd64
 
-Tags: 8-jre-1.7.5-jaxrs
-Directory: 8-jre-1.7.5-jaxrs
+Tags: 8-jre-7.0.7-plume, 7.0.7-plume
+Directory: TomEE-7.0/jre8/plume
 
-Tags: 8-jre-1.7.5-plume
-Directory: 8-jre-1.7.5-plume
+Tags: 8-jre-7.0.7-plus, 7.0.7-plus
+Directory: TomEE-7.0/jre8/plus
 
-Tags: 8-jre-1.7.5-plus
-Directory: 8-jre-1.7.5-plus
+Tags: 8-jre-7.0.7-webprofile, tomee7.0
+Directory: TomEE-7.0/jre8/webprofile
 
-Tags: 8-jre-1.7.5-webprofile
-Directory: 8-jre-1.7.5-webprofile
+Tags: 8-jre-7.0.7-microprofile, 7.0.7-microprofile
+Directory: TomEE-7.0/jre8/microprofile
 
-Tags: 8-jre-7.0.4-plume
-Directory: 8-jre-7.0.4-plume
+Tags: 8-jre-7.1.2-plume, 7.1.2-plume
+Directory: TomEE-7.1/jre8/plume
 
-Tags: 8-jre-7.0.4-plus
-Directory: 8-jre-7.0.4-plus
+Tags: 8-jre-7.1.2-plus, 7.1.2-plus
+Directory: TomEE-7.1/jre8/plus
 
-Tags: 8-jre-7.0.4-webprofile
-Directory: 8-jre-7.0.4-webprofile
+Tags: 8-jre-7.1.2-microprofile, 7.1.2-microprofile
+Directory: TomEE-7.1/jre8/microprofile
 
-Tags: 8-jre-7.0.5-plume
-Directory: 8-jre-7.0.5-plume
+Tags: 8-jre-7.1.2-webprofile, 7.1.2-webprofile, tomee7, tomee7.1
```

Comment:
`tomee8`, `tomee7`, `tomee7.1`, and `tomee7.0` seem like strange tags since they will be redundant from the user perspective (`tomee:tomee8`).  Would these be better as just `8`, `7.1`, etc?

Reply:
Agreed. And fixed.